src/client/components/FavouriteList/FavouriteList.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { action } from '@storybook/addon-actions';
+import './FavouriteList.styles.css';
+
+export default function FavouriteList({ productList }) {
+  const [counter, setCounter] = useState(0);
+  const handleClick1 = () => {
+    setCounter(counter + 1);
+  };
+
+  const handleClick2 = () => {
+    setCounter(counter - 1);
+  };
+
+  return (
+    <>
+      <h2>Favourites</h2>
+      <ul className="container">
+        {productList.map((product) => {
```

Comment:
Preferrably dont nest logic within the jsx ( the productList mapping ). It makes it harder to read.
Image you open a component and you have 5 seconds to read and understand what it does.